source/lib/JMSLab/builders/build_latex.py

```diff
@@ -44,3 +46,40 @@ def add_call_args(self):
                                   os.path.normpath(self.log_file))
         self.call_args = args
         return None
+     
+    def add_bib_name(self, target):
+        if bool(target):
+            target = misc.make_list_if_string(target)
+            target_file = str(target[0]).split(".")[0]
```

Comment:
I think this wouldn't work if a file name contains multiple dots.

Reply:
@rcalvo12 Thanks! Was it important to have the `target = misc.make_list_if_string(target)` line? Unsure of the answer but flagging that this also took that out. Otherwise looks good.